lbrynet/lbryfilemanager/EncryptedFileManager.py

```diff
@@ -108,13 +120,14 @@ def _start_lbry_files(self):
         yield self._check_stream_info_manager()
         lbry_files_and_options = yield self._get_all_lbry_files()
         for rowid, stream_hash, options in lbry_files_and_options:
+            lbry_file = yield self.start_lbry_file(rowid, stream_hash, payment_rate_manager,
+                                                   blob_data_rate=options)
             try:
-                lbry_file = yield self.start_lbry_file(rowid, stream_hash, payment_rate_manager,
-                                 blob_data_rate=options)
                 yield lbry_file.restore()
             except Exception as err:
-                msg = "Failed to start lbry file, rowid: %i, stream_hash: %s, options: %s, err: %s"
-                log.error(msg, rowid, stream_hash, options, err)
+                log.error("Failed to start stream: %s, error: %s", stream_hash, err)
```

Comment:
this is better as `log.exception("Failed to start stream: %s", stream_hash)`

https://docs.python.org/2/library/logging.html#logging.Logger.exception